### ISSUE TITLE
Error detection for `<rp>` element in permitted-contents

### ISSUE THREAD

**Issue by yamanoku** [markuplint/markuplint#637]
- Markuplint version: `3.0.0`
- Parser lang: HTML
- Node.js version: `19.x`
- OS: macOS

## Describe the bug

`<ruby>` with `<rp>` is detected as an error by markuplint.

- [No error in v2.11.4](https://github.com/yamanoku-playground/2022-12-25-markuplint-ruby-rp-element/tree/branch-2-11-4)
- [Nu Html Checker](https://validator.w3.org/nu/) didn't give me any error.

## Code Example

```html
<div>
  <ruby>漢<rp>（</rp><rt>かん</rt><rp>）</rp>字<rp>（</rp><rt>じ</rt><rp>）</rp></ruby>
</div>
```

This code sample is from the WHATWG HTML Standard.
https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-rp-element

## Steps To Reproduce

1. `git clone https://github.com/yamanoku-playground/2022-12-25-markuplint-ruby-rp-element.git`
2. `npm ci`
3. `npm run html:lint`

## The current behavior

```shell
<markuplint> error: 要素が必要です。(「rp」が必要です) (permitted-contents) /Users/yamanoku/works/_sandbox/2022-12-25-markuplint-ruby-rp-element/index.html:9:9
  8: ••••••••<h1>rp要素のmarkuplint用デモ</h1>
  9: ••••••••<ruby>漢<rp>（</rp><rt>かん</rt><rp>）</rp>字<rp>（</rp><rt>じ</rt><rp>）</rp></ruby>
```

## The expected behavior

Not detected as error

**Comment by YusukeHirao** [comment]
@yamanoku Thank you for your report.
I'm surprised you noticed the problem. 

But it may be a long battle.

**Comment by yamanoku** [comment]
Thanks !

I reverted it to reflect the update. https://togithub.com/yamanoku/yamanoku.github.io/pull/1070
